.gitignore

```diff
@@ -117,3 +117,4 @@ chains/era/configs/*
 configs/*
 era-observability/
 core/tests/ts-integration/deployments-zk
+transactions/
```

Comment:
There should be newline here

Reply:
Addressed in https://github.com/matter-labs/zksync-era/pull/2787/commits/2915f9f65fc8fd881b2d638e3eb17a3bc808320e